smoketest/tests/create_agent.rs

```diff
@@ -0,0 +1,29 @@
+use snowbridge_smoketest::contracts::i_gateway::AgentCreatedFilter;
+use snowbridge_smoketest::helper::*;
+use snowbridge_smoketest::parachains::bridgehub::api::ethereum_control::events::CreateAgent;
+use snowbridge_smoketest::xcm::construct_xcm_message;
+
+#[tokio::test]
+async fn create_agent() {
+    let test_clients = initial_clients().await.expect("initialize clients");
+
+    let message = construct_xcm_message(
+        construct_create_agent_call(&test_clients.bridge_hub_client)
+            .await
+            .expect("construct innner call."),
+    );
+
+    let result = send_xcm_transact(&test_clients.template_client, message)
```

Comment:
I would prefer to avoid sudo operation from relaychain and let parachain self-governing to create the channel. Btw, the new parachain is also a pre-step for the ongoing arbitratry transact tests.